includes/admin/class-wc-admin-addons.php

```diff
@@ -722,11 +788,11 @@ public static function install_woocommerce_payments_addon( $section = '_featured
 	 */
 	public static function show_extension( $item ) {
 		$location = WC()->countries->get_base_country();
-		if ( isset( $item->geowhitelist ) && ! in_array( $location, $item->geowhitelist, true ) ) {
+		if ( isset( $item->geo_allow_list ) && ! in_array( $location, $item->geo_allow_list, true ) ) {
```

Comment:
hi, sorry, i did not realize that these naming are also being used here as well, which is being used for other items as well and not just wc pay. we should revert this change, because we don't know if all the other promotional extensions have changed the wording as well, and they have not then these wordings will not work for them anymore.